src/vm/core.zig

```diff
@@ -439,7 +471,11 @@ test "update pc regular with imm" {
     instruction.op_1_addr = instructions.Op1Src.Imm;
     const operands = OperandsResult.default();
     // Create a new VM instance.
-    var vm = try CairoVM.init(&allocator);
+    var config = Config{
```

Comment:
can you add a `default` function in `Config` and use it in tests ?

Reply:
right thats much better

Reply:
As far as I recall you can also set defaults in the struct itself here:

https://github.com/keep-starknet-strange/cairo-zig/blob/3abd1bdfae155497ec9f46c8ff88f1b0b46c2183/src/vm/config.zig#L6-L8

Like so:

```zig
pub const Config = struct { 
     proof_mode: bool = false, 
     enable_trace: bool = false, 
```

and pass `.{}` everywhere:
```zig
var vm = try CairoVM.init(&allocator, .{});
```

Reply:
interesting!